compose/common.yml

```diff
@@ -235,6 +235,20 @@ services:
     ports:
       - ${STAKE_POOL_API_PORT:-4010}:3000
 
+  handle-provider-server:
+    <<:
+      - *from-sdk
+      - *logging
+      - *provider-server
+      - *with-postgres
+    environment:
+      <<:
+        - *sdk-environment
+        - *provider-server-environment
+      SERVICE_NAMES: handle
+    ports:
+      - ${HANDLE_API_PORT:-4020}:3000
```

Comment:
I would use a progressive port number here: 4011 rather than 4020

But moreover `handle-provider-server` specifications are missing in `packages/e2e/docker-compose.yml` and in `packages/cardano-services/docker-compose.yml`.
They are required to provide the handle policyIds from the env.

Reply:
Fixed on 5eef2a2